spec/mailers/user_mailer_spec.rb

```diff
@@ -40,17 +40,23 @@
 
   describe 'reservation_status_update' do
     it 'sends to the reserver' do
+      # make it a request so the email sends
+      @res.update_attributes(flags: Reservation::FLAGS[:request])
```

Comment:
Can't we move this into the `before` block above after `@res` is defined? At least that will cut down on some duplication. We should make sure to move the comment as well so it's easy to understand *why* we're flagging it as a request.